container/nginx/confs/non_tls_proxy.conf

```diff
@@ -6,3 +6,14 @@ server {
     include /etc/nginx/conf.d/register.conf;
     include /etc/nginx/conf.d/shared.conf;
 }
+
+server {
+    listen 80;
```

Comment:
Will never arrive here: https://github.com/protocol/bifrost-infra/commit/d700bfeba3a26be72c7ef2ebc864e2c0635ceb2c#diff-a114c9e56a17ead31a0d56c0894354aa74791166e6bc1ca44df5d19a55b70888R159

Reply:
Ya this is just for local dev, when L1 is in non-TLS mode.